sdcflows/utils/wrangler.py

```diff
@@ -232,57 +258,74 @@ def find_estimators(*, layout, subject, fmapless=True, force_fmapless=False):
 
     estimators = []
 
-    # Set up B0 fieldmap strategies:
-    for fmap in layout.get(suffix=["fieldmap", "phasediff", "phase1"], **base_entities):
-        e = fm.FieldmapEstimation(
-            fm.FieldmapFile(fmap.path, metadata=fmap.get_metadata())
-        )
-        estimators.append(e)
-
-    # A bunch of heuristics to select EPI fieldmaps
-    sessions = layout.get_sessions()
-    acqs = tuple(layout.get_acquisitions(suffix="epi") + [None])
-    contrasts = tuple(layout.get_ceagents(suffix="epi") + [None])
+    b0_ids = tuple()
+    with suppress(BIDSEntityError):
+        b0_ids = layout.get_B0FieldIdentifiers()
```

Comment:
TIL about this convenience function. We probably want to ensure we are only concerned with the current subject.

```suggestion
        b0_ids = layout.get_B0FieldIdentifiers(**base_entities)
```